### PR TITLE
Sign up rework/fe

### PR DESCRIPTION
**Pull request by tauanlongaretti**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Change Status

- [X] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [X] Made use of localhost to test whether both success and failure messages are working depending if email is posted successfully or not to the back-end. 

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] My changes generate no new warnings
- [X] There are no merge conflicts

### CODE REVIEW

**Comment by tauanlongaretti** [comment]
[Success-Failure Feature / Sign-Up Rework](https://trello.com/c/xnu4HCba/87-success-failure-feature-sign-up-rework)

File: src/actions/houseHoldActions.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const INVITE_MEMBER = 'INVITE_MEMBER';
 const fetchHousehold = () => (dispatch) => {
   dispatch({ type: LOADING });
   return axiosWithAuth()
-    .get('/members/household/assignable')
+    .get('/members/household')
```
Comment (BenHall-7) [inline src/actions/houseHoldActions.js:13]: Either the backend will need to be updated before we merge this or we can remove this change temporarily

Comment (tauanlongaretti) [reply src/actions/houseHoldActions.js:13]: It can wait then. I only added it because you asked me to.

Comment (Adammonast) [reply src/actions/houseHoldActions.js:13]: I'm going to go ahead and approve, but we'll hold off on merging at the moment

**Pull request merged**